src/Nest/CrossPlatform/TypeExtensions.cs

```diff
@@ -16,6 +18,7 @@ internal static bool IsGeneric(this Type type)
 #endif
 		}
 
+		[MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Curious what triggered this change?

Reply:
I should have looked at the IL before making this change....it appears to make no difference. Will revert 👍

Reply:
reverted